devicetypes/smartthings/zwave-switch-secure.src/zwave-switch-secure.groovy

```diff
@@ -22,9 +22,7 @@ metadata {
 
 		fingerprint inClusters: "0x25,0x98"
 		fingerprint deviceId: "0x10", inClusters: "0x98"
-		fingerprint mfr: "0086", prod: "0003", model: "008B", deviceJoinName: "Aeon Labs Nano Switch"
-		fingerprint mfr: "0086", prod: "0103", model: "008B", deviceJoinName: "Aeon Labs Nano Switch"
-		fingerprint mfr: "027A", prod: "A000", model: "A001", deviceJoinName: "Zooz ZEN26 Switch"
+		fingerprint mfr: "0152", prod: "0003", model: "0002", deviceJoinName: "iTec Home Light Switch"
```

Comment:
@iTecHome can you please change deviceJoinName to `iTec Switch`?

Reply:
@PKacprowiczS we will be releasing other switches later in the year so need to identify this as it is to avoid confusion

Reply:
We'e changed our device join name standards to make them more uniform.